src/components/ui/Pressable/Pressable.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import Pressable from './Pressable';
+
+export default {
+  title: 'Ui/Pressable',
+  component: Pressable,
+  argTypes: {},
+} as ComponentMeta<typeof Pressable>;
+
+const Template: ComponentStory<typeof Pressable> = (args) => (
```

Comment:
`Pressable` has some additional styles for `p/h3/h5` so let's add one more story with children wrapped with `h3` for example